components/style/properties/longhand/box.mako.rs

```diff
@@ -351,57 +351,23 @@ ${helpers.single_keyword("overflow-clip-box", "padding-box content-box",
     spec="Internal, not web-exposed, \
           may be standardized in the future (https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-clip-box)")}
 
+<% overflow_custom_consts = { "-moz-scrollbars-horizontal": "SCROLLBARS_HORIZONTAL",
+                              "-moz-scrollbars-vertical": "SCROLLBARS_VERTICAL",
+                              "-moz-hidden-unscrollable": "CLIP" }
+%>
 // FIXME(pcwalton, #2742): Implement scrolling for `scroll` and `auto`.
 ${helpers.single_keyword("overflow-x", "visible hidden scroll auto",
-                         extra_gecko_values="clip",
                          need_clone=True, animation_value_type="none",
+                         extra_gecko_aliases="-moz-scrollbars-none=hidden",
+                         extra_gecko_values="-moz-scrollbars-horizontal -moz-scrollbars-vertical -moz-hidden-unscrollable",
```

Comment:
I don't think this is quite right.  Gecko will parse `-moz-scrollbars-horizontal` as a value for the shorthand, but it gets turned into `overflow-x: scroll; overflow-y: hidden;`, and similarly for `-moz-scrollbars-vertical`.